frontend/src/pages/ShootItemTerminal.vue

```diff
@@ -15,737 +15,225 @@ limitations under the License.
-
-  const connectTimeoutSeconds = 5
-  await pTimeout(connectPromise, connectTimeoutSeconds * 1000, `Could not connect within ${connectTimeoutSeconds} seconds`)
-
-  const podRunningPromise = new Promise((resolve, reject) => {
-    const resolveOnce = value => {
-      ws.removeEventListener('message', messageHandler)
-      ws.removeEventListener('close', closeHandler)
-
-      resolve(value)
-    }
-    const rejectOnce = reason => {
-      ws.removeEventListener('message', messageHandler)
-      ws.removeEventListener('close', closeHandler)
-
-      reject(reason)
-    }
-
-    const closeHandler = error => {
-      rejectOnce(error)
-    }
-    const messageHandler = ({ data: message }) => {
-      let event
-      try {
-        event = JSON.parse(message)
-      } catch (error) {
-        console.error('could not parse message')
-        return
-      }
-      const pod = event.object
-      if (typeof handleEvent === 'function') {
-        try {
-          handleEvent(event)
-        } catch (error) {
-          console.error('error during handleEvent', error.message)
-        }
-      }
-
-      const phase = get(pod, 'status.phase')
-      if (includes(['Failed', 'Succeeded'], phase)) {
-        rejectOnce(new Error(`Pod is in phase ${phase}`))
-        return
-      } else if (event.type === 'DELETED') {
-        rejectOnce(new Error('Pod deleted'))
-        return
-      }
-
-      const containerStatuses = get(pod, 'status.containerStatuses')
-      const terminalContainerStatus = find(containerStatuses, ['name', containerName])
-      const isContainerReady = get(terminalContainerStatus, 'ready', false)
-
-      if (phase === 'Running' && isContainerReady) {
-        resolveOnce()
-      }
-    }
-    ws.addEventListener('message', messageHandler)
-    ws.addEventListener('close', closeHandler)
-  })
-  return pTimeout(podRunningPromise, timeoutSeconds * 1000, `Timed out after ${timeoutSeconds}s`)
-}
-
-function getDetailedConnectionStateText (terminalContainerStatus) {
-  const state = get(terminalContainerStatus, 'state')
-  const stateKeys = intersection(['waiting', 'running', 'terminated'], keys(state))
-  const stateType = head(stateKeys)
-
-  let text = ''
-  if (!stateType) {
-    return text
-  }
-
-  text = `Container is ${stateType}`
-
-  const reason = get(terminalContainerStatus, ['state', stateType, 'reason'])
-  if (!reason) {
-    return text
-  }
-
-  return `${text}: ${reason}`
+import cloneDeep from 'lodash/cloneDeep'
+import difference from 'lodash/difference'
+import GSplitpane from '@/components/GSplitpane'
+import GTerminal from '@/components/GTerminal'
+import TargetSelectionDialog from '@/dialogs/TargetSelectionDialog'
+import { TargetEnum } from '@/utils'
+import { shootItem } from '@/mixins/shootItem'
+import { listTerminalSessions } from '@/utils/api'
+import { GSymbolTree, TreeItem, PositionEnum } from '@/lib/g-symbol-tree'
+
+import 'splitpanes/dist/splitpanes.css'
+
+Vue.use(require('vue-shortkey'))
```

Comment:
Use directive directly not as global plugin

Reply:
not possible as directive not exported. However I will move it to `main.js`